blocks/card-social-comment/src/CardSocialComment.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import type { Meta, StoryObj } from "@storybook/react";
+import storyDialog from "../../../.storybook/decorators/storyDialog";
+import Usage from "../usage.mdx";
+import CardSocialComment from "./CardSocialComment";
+
+const meta = {
+  title: "Card/Social/Comment",
+  component: CardSocialComment,
+  parameters: {
+    layout: "centered",
+    githubUsername: "", // (optional) Your github username. If provided, your avatar will be displayed in the story toolbar
```

Comment:
If you like, you can add your github username here to show it in the storybook.

<img width="303" alt="image" src="https://github.com/siriwatknp/joy-treasury/assets/18292247/dd188671-1a17-4d3e-8045-38db3fb4e665">